### PR TITLE
This should never have been there.

### PR DESCRIPTION
**Pull request by jaheba**

### CODE REVIEW

File: src/trace/optimisation.rs [hunk 1]
```diff
@@ -18,6 +18,7 @@ where F: Fn(&OpCode) -> bool {
 }
 
 
+
```
Comment (ltratt) [inline src/trace/optimisation.rs:21]: I think two blank lines is enough, and a third isn't needed.

**Comment by jaheba** [comment]
Sorry for the mess

**Comment by ltratt** [comment]
Except in very, very special cases (which are extremely rare), unwanted code should be deleted, not commented out.

**Comment by jaheba** [comment]
Agreed.

**Comment by ltratt** [comment]
I think this PR is old enough that we can close it.

**Pull request closed without merging**